Libraries/LibWeb/WebGPU/GPUAdapter.idl

```diff
@@ -0,0 +1,28 @@
+#import <WebGPU/GPUAdapterInfo.idl>
+#import <WebGPU/GPUDevice.idl>
+#import <WebGPU/GPUSupportedFeatures.idl>
+#import <WebGPU/GPUSupportedLimits.idl>
+
+// https://www.w3.org/TR/webgpu/#enumdef-gpupowerpreference
+enum GPUPowerPreference {
+    "low-power",
+    "high-performance",
+};
+
+// https://www.w3.org/TR/webgpu/#dictdef-gpurequestadapteroptions
+dictionary GPURequestAdapterOptions {
+    DOMString featureLevel = "core";
+    GPUPowerPreference powerPreference;
+    boolean forceFallbackAdapter = false;
+    // FIXME: Support xrCompatible
```

Comment:
Shouldn't unsupported fields be written as:
```
[FIXME] boolean xrCompatible;
```

and methods in their respective FIXME way?